antora.yml

```diff
@@ -1,6 +1,7 @@
 name: tinymce
 title: TinyMCE Documentation
-version: '6'
+version: 'latest'
```

Comment:
@MitchC1999 thinking in the near future, does this mean that once this is merged we will need to change the way we create branches such as `feature/latest/DOC-<num>` rather than `feature/<ver-number>/DOC-<num>`

Reply:
No, the only change that will need to be made will be in the `feature_6_docs.yml` file, which we will need to update for 7 anyway. I'll be collating a list of items to update in the docs to prepare for that release.